add_car.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+require_once "vendor/autoload.php";
+require_once "src/utils.php";
+
+use CarStore\UpdateCars;
+use CarStore\Car;
+
+session_start();
+
+if (
+    isset($_POST["name"]) &&
+    isset($_POST["year_made"]) &&
+    isset($_POST["zero_sixty"]) &&
+    isset($_POST["price"]) &&
+    isset($_POST["brand"])
+) {
+    $name = $_POST["name"];
+    $year_made = $_POST["year_made"];
+    $zero_sixty = $_POST["zero_sixty"];
+    $price = $_POST["price"];
+    $brand = $_POST["brand"];
+
+    $car = new Car($name, $year_made, $zero_sixty, $price, $brand);
+
+    $user = new UpdateCars(make_db());
+    $user->addCar($car);
```

Comment:
See my comment about making addCar return a boolean, check for success here and display a generic (please try again type deal) message if it fails for whatever reason. 

You probably won't be able to make it fail easily, but it's always worth handling failure just to be safe